internal/controlplane/handlers_repositories.go

```diff
@@ -368,6 +368,25 @@ func (s *Server) ListRemoteRepositoriesFromProvider(
 			continue
 		}
 
+		registeredRepos, err := s.store.ListRepositoriesByProjectID(ctx, db.ListRepositoriesByProjectIDParams{
+			ProjectID: projectID,
+			Provider:  getNameFilterParam(provider.Name),
+		})
+		if err != nil {
+			// Here we try to handle errors gracefully by
+			// simply not filtering.
```

Comment:
I would love to hear opinions on this.

Reply:
This seems like the right approach to me

Reply:
Concern: Image a customer calls this endpoint in a script and uses the registered/not registered information for some purpose, such as selecting new repos to register or something like this. If we have a transient failure, their script could end up treating some registered repos as unregistered, and this could cause problems.

I'm not convinced that swallowing this error is helpful, and it could cause problems for the client.

Reply:
Yes, that's what I was concerned with as well, but as far as I can tell the only problem that can occur is that a Customer tries to `minder repo register` the same repo again, and at that point either it fails completely or it succeeds and returns a "repo already registered" message.

Besides, if a Customer uses this endpoint as last interaction with Minder before triggering his own internal processes (e.g. for some sort of reporting feature or so), then he is probably misusing it and we might need a separate endpoint just for such use cases.

It's not a big change either way and we might want to fail unconditionally to keep it simple for now, let me know what you folks prefer.

Reply:
I don't feel strongly about this. I'm fine with propagating the error to the caller as well.

Reply:
After discussing this, we decided to fail rather than hiding the error.